test/case/ietf_syslog/remote/topology.dot

```diff
@@ -1,32 +1,30 @@
 graph "2x2" {
 	layout="neato";
 	overlap="false";
-	esep="+20";
+	esep="+60";
 
         node [shape=record, fontname="monospace"];
 	edge [color="cornflowerblue", penwidth="2"];
 
 	host [
-	    label="host | { <mgmt1> mgmt1 | <data1> data1 | <mgmt2> mgmt2 | <data2> data2 }",
+	    label="host | { <cli_mgmt> cli_mgmt | <cli_data> cli_data | <ser_mgmt> ser_mgmt | <ser_data> ser_data }",
 	    pos="0,12!",
 	    kind="controller",
 	];
 
-        target1 [
-	    label="{ <mgmt> mgmt | <data> data | <target2> target2} | target1",
-	    pos="10,18!",
+        client [
+	    label="{ <mgmt> mgmt | <data> data | <server> server} | client",
```

Comment:
Name the port to_server instead, to seperate it from the dut server, same on the other dut.

Reply:
good point